.github/workflows/build.yml

```diff
@@ -0,0 +1,48 @@
+name: LazyDFU build script
```

Comment:
We have a long-established job I set up on the [Velocity Jenkins](https://ci.velocitypowered.com/job/lazydfu/). GitHub Actions isn't necessary.

Reply:
I left Gradle Wrapper Validation for security reasons if someone wanted to replace it with malware